pdns/iputils.hh

```diff
@@ -59,6 +59,25 @@
 #define le64toh(x) OSSwapLittleToHostInt64(x)
 #endif
 
+#ifdef BE_64
```

Comment:
I suppose you could try to check for some flavor of __sun__, __sun, etc. instead... but (1) I don't remember the magic define, and (2) it's ugly anyway.